tests/test_Producer.py

```diff
@@ -22,6 +23,14 @@ def test_basic_api():
 
     p.produce('mytopic')
     p.produce('mytopic', value='somedata', key='a key')
+    p.produce('mytopic', value='somedata', key='a key', headers=[('headerkey', 'headervalue')])
+    p.produce('mytopic', value='somedata', key='a key', headers=[('dupkey', 'dupvalue'), ('dupkey', 'dupvalue')])
```

Comment:
have the second dupvalue be something different to make sure they are actually treated as different entities